uproot/containers.py

```diff
@@ -792,19 +792,45 @@ def read(self, chunk, cursor, context, file, selffile, parent, header=True):
         else:
             is_memberwise = False
 
+        # note: self._values can also be a NumPy dtype, and not necessarily a class (e.g. type(self._values)==type)
+        _value_typename = _content_typename(self._values)
         if is_memberwise:
-            raise NotImplementedError(
-                """memberwise serialization of {0}
-in file {1}""".format(
-                    type(self).__name__, selffile.file_path
+            # let's hard-code in logic for std::pair<T1,T2> for now
+            if not _value_typename.startswith("pair"):
+                raise NotImplementedError(
+                    """memberwise serialization of {0}({1})
+    in file {2}""".format(
+                        type(self).__name__, _value_typename, selffile.file_path
+                    )
                 )
-            )
 
-        length = cursor.field(chunk, _stl_container_size, context)
+            # there's extra stuff, maybe?
+            _num_memberwise_bytes = cursor.field(chunk, _stl_container_size, context)
+            _something_else = cursor.field(chunk, struct.Struct(">H"), context)
```

Comment:
This portion that we're at right now looks like

```
--+---+---+---+---+---+---+---+---+---+
  0   0   0 215 190 210   0   0   0  13  
--- --- --- --- --- --- --- --- --- --- 
```

Unclear what the first 6 bytes are here, but the last 4 seem to be number of elements in the vector (`length`) in the line below. I assumed it might be 4 bytes / 2 bytes but still not sure what they're meant to match up with.